### PR TITLE
Update Project Zomboid Egg to support beta versions

### PR DESCRIPTION
**Pull request by BASIC-BIT**
These were my edits to get this egg working for the "b41multiplayer" beta version of Project Zomboid:
- Consume the SteamCMD script and add associated variables
- Update image to support "b41multiplayer" beta branch

### All Submissions:

* [X] Have you followed the guidelines in our Contributing document?
* [X] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [X] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

### Changes to an existing Egg:

1. [X] Have you added an explanation of what your changes do and why you'd like us to include them?
2. [X] Have you tested your Egg changes?

### CODE REVIEW

**Comment by Software-Noob** [comment]
You should take a look at our existing steamcmd script. Your current variable will cause the docker entrypoint to overwrite the beta branch on auto update during boot, because your beta param doesn't get passed to it

**Comment by BASIC-BIT** [comment]
Thank you for the suggestion!  I've updated the egg to use the steamcmd script - let me know if anything else needs changes!

File: game_eggs/steamcmd_servers/project_zomboid/egg-project-zomboid.json [hunk 1]
```diff
@@ -28,44 +33,125 @@
...
+            "name": "Steam Auth Code",
+            "description": "Steam user setup. If a user has 2fa enabled it will most likely fail due to timeout. Leave blank for anon install.",
+            "env_variable": "STEAM_AUTH",
+            "default_value": "",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "nullable|string"
+        },
+        {
+            "name": "WINDOWS_INSTALL",
```
Comment (Software-Noob) [inline game_eggs/steamcmd_servers/project_zomboid/egg-project-zomboid.json:104]: Please don't add variables that are not necessary. Having many variables makes it messy, and this one, for example, will break the whole installation if someone sets it to 1.

zomboid doesn't need steam user or branch password to download it. Thus, all the steam variables are not required.

**Comment by Wirpy** [comment]
Hope this gets merged soon! B41 beta is great.

**Review by Software-Noob — APPROVED** [review]
I've fixed the rest myself. LGTM, thank you for contributing :+1:

**Pull request merged**